analytics_data_api/v0/views/programs.py

```diff
@@ -0,0 +1,63 @@
+from django.db.models import Q
+
+from analytics_data_api.v0 import models, serializers
+from analytics_data_api.v0.views import APIListView
+
+
+class ProgramsView(APIListView):
+    """
+    Returns metadata information for programs.
+
+    **Example Request**
+
+        GET /api/v0/course_programs/?program_ids={program_id},{program_id}
+
+    **Response Values**
+
+        Returns metadata for every program:
+
+            * program_id: The ID of the program for which data is returned.
+            * program_type: The type of the program
+            * program_title: The title of the program
+            * created: The date the counts were computed.
+
+    **Parameters**
+
+        Results can be filed to the course IDs specified or limited to the fields.
+
+        program_ids -- The comma-separated program identifiers for which metadata is requested.
+            Default is to return all programs.
+        fields -- The comma-separated fields to return in the response.
+            For example, 'program_id,created'.  Default is to return all fields.
+        exclude -- The comma-separated fields to exclude in the response.
+            For example, 'course_id,created'.  Default is to not exclude any fields.
+    """
+    always_exclude = ['course_id']  # original model has course_id, but the serializer does not (after aggregation)
```

Comment:
Can we just remove this field from the serializer entirely or do we need it to keep django happy?